### PR TITLE
Korean translation update

### PR DESCRIPTION
**Pull request by jerrybeomsoo**
Updated string.js file to include Korean translations of password change, search function, and etc.

### CODE REVIEW

File: public/src/js/strings.js [hunk 1]
```diff
@@ -846,73 +852,86 @@ var translations = {
...
+				tw: "聲音延遲校正",
+				ko: "오디오 레이턴시 조절"
 				
 			},
 			content: {
 				ja: "背景で鳴っている音を聴いてみましょう。\n\n音が聞こえたら、太鼓の面（%sまたは%s）をたたこう！",
 				en: "Listen to a sound playing in the background.\n\nHit the surface of the drum (%s or %s) as you hear it!",
-				tw: "仔細聆聽背景播放的音效。\n\n聽到音效就敲打鼓面 (%s 或 %s)！"
+				tw: "仔細聆聽背景播放的音效。\n\n聽到音效就敲打鼓面 (%s 或 %s)！",
+				ko: "소리에 맞춰 북(%s나 %s)을 쳐주세요!"
```
Comment (KatieFrogs) [inline public/src/js/strings.js:888]: Please make this string more interesting, it should not be this short and serious. Also I think there is a space missing after the first `%s`, those get replaced with buttons, by default `F` and `J`

File: public/src/js/strings.js [hunk 2]
```diff
@@ -846,73 +852,86 @@ var translations = {
...
 			},
 			contentAlt: {
 				ja: "背景で鳴っている音を聴いてみましょう。\n\n音が聞こえたら、太鼓の面をたたこう！",
 				en: "Listen to a sound playing in the background.\n\nHit the surface of the drum as you hear it!",
-				tw: "仔細聆聽背景播放的音效。\n\n聽到音效就敲打鼓面！"
+				tw: "仔細聆聽背景播放的音效。\n\n聽到音效就敲打鼓面！",
+				ko: "소리에 맞춰 북을 쳐주세요!"
```
Comment (KatieFrogs) [inline public/src/js/strings.js:894]: Same as above

**Review by KatieFrogs — CHANGES_REQUESTED** [review]

File: public/src/js/strings.js (hunk 1 above)
Comment (jerrybeomsoo) [reply public/src/js/strings.js]: Fixed. The string to have more similar meaning to original `en` string, plus removed the space.

File: public/src/js/strings.js (hunk 2 above)
Comment (jerrybeomsoo) [reply public/src/js/strings.js]: Fixed.

File: public/src/js/strings.js [hunk 3]
```diff
@@ -846,73 +852,86 @@ var translations = {
...
-				tw: "延遲校正結果"
+				tw: "延遲校正結果",
+				ko: "레이턴시 조절 결과"
 			},
 			content: {
 				ja: "オーディオ・レイテンシー: %s\nビデオ・レイテンシ: %s\n\nこれらのレイテンシーの値は、設定で設定することができます。",
 				en: "Audio latency: %s\nVideo latency: %s\n\nYou can configure these latency values in the settings.",
-				tw: "聲音延遲 :%s\n畫面延遲: %s\n\n您可以在設定中調整這些數值。"
+				tw: "聲音延遲 :%s\n畫面延遲: %s\n\n您可以在設定中調整這些數值。",
+				ko: "오디오 레이턴시: %s\n 레이턴시: %s\n설정에서 레이턴시 값들을 조절할 수 있습니다."
```
[the remark this answers is not in the document]
Comment (jerrybeomsoo) [reply public/src/js/strings.js]: Fixed the space to `비디오`, and changed `\n` to `\n\n`. Presumably the word `비디오` has been replaced to space during the editing process.

File: public/src/js/strings.js [hunk 4]
```diff
@@ -1078,96 +1105,118 @@ var translations = {
...
-			tw: "目前密碼不符合"
+			tw: "目前密碼不符合",
+			ko: "기존 비밀번호가 틀립니다."
 		},
 		invalid_new_password: {
 			ja: "このパスワードは使用できません。新しいパスワードが6文字以上であることを確認してください",
 			en: "Cannot use this password, please check that your new password is at least 6 characters long",
-			tw: "無法使用此密碼，您的新密碼長度至少要 6 個字"
+			tw: "無法使用此密碼，您的新密碼長度至少要 6 個字",
+			ko: "해당 비밀번호를 사용할 수 없습니다. 비밀번호가 6자를 넘어가도록 설정하시기 바랍니다."
```
[the remark this answers is not in the document]
Comment (jerrybeomsoo) [reply public/src/js/strings.js]: Fixed.

File: public/src/js/strings.js [hunk 5]
```diff
@@ -1385,6 +1455,18 @@ var translations = {
...
+			],
+			ko: [
+				"CTRL+F를 눌러 검색 창을 여세요!",
+				"자유롭게 필터를 조합해 검색하세요!",
+				"\"genre:\" 키워드로 원하는 장르의 곡만 찾아보세요! (예시: \"genre:variety\", \"genre:namco\")",
+				"\"oni:10\" 같은 키워드로 원하는 난이도를 가진 곡을 찾아보세요!",
+				"\"ura:1-5\" 같은 키워드로 여러 난이도를 선택할 수 있어요!",
+				"풀 콤보한 곡을 찾아보고 싶나요? \"gold:any\", \"gold:oni\" 등의 키워드로 검색할 수 있습니다!",
+				"창작 채보 곡들을 검색하고 싶나요? \"creative:yes\" 키워드를 사용하세요!",
+				"\"lyrics:yes\" 키워드로 가사가 있는 곡들을 선택할 수 있어요!!",
```
[the remark this answers is not in the document]
Comment (jerrybeomsoo) [reply public/src/js/strings.js]: Fixed. Thank you!

**Pull request merged**